### PR TITLE
Added comments to improve clarity of samples

### PR DESCRIPTION
**Pull request by shubha-rajan**
Added comments to improve the clarity of Cloud SQL samples by showing example values for environment variables.

### CODE REVIEW

File: cloud-sql/mysql/Startup.cs [hunk 1]
```diff
@@ -123,10 +123,11 @@ DbConnection NewMysqlTCPConnection()
                 // Remember - storing secrets in plaintext is potentially unsafe. Consider using
                 // something like https://cloud.google.com/secret-manager/docs/overview to help keep
                 // secrets secret.
-                Server = Environment.GetEnvironmentVariable("DB_HOST"),
-                UserID = Environment.GetEnvironmentVariable("DB_USER"),
-                Password = Environment.GetEnvironmentVariable("DB_PASS"),
-                Database = Environment.GetEnvironmentVariable("DB_NAME"),
+                Server = Environment.GetEnvironmentVariable("DB_HOST"),   // e.g. '127.0.0.1'
+                                // Set Host to 'cloudsql' when deploying to App Engine Flexible environment
```
Comment (amanda-tarafa) [inline cloud-sql/mysql/Startup.cs:127]: Nit: this line's indent is off

File: cloud-sql/mysql/Startup.cs [hunk 2]
```diff
@@ -144,19 +145,19 @@ DbConnection NewMysqlUnixSocketConnection()
             // [START cloud_sql_mysql_dotnet_ado_connection_socket]
             // Equivalent connection string: 
             // "Server=<dbSocketDir>/<INSTANCE_CONNECTION_NAME>;Uid=<DB_USER>;Pwd=<DB_PASS>;Database=<DB_NAME>;Protocol=unix"
-            String dbSocketDir = Environment.GetEnvironmentVariable("DB_SOCKET_PATH") ?? "/cloudsql";
-            String instanceConnectionName = Environment.GetEnvironmentVariable("INSTANCE_CONNECTION_NAME");
+            String dbSocketDir = Environment.GetEnvironmentVariable("DB_SOCKET_PATH") ?? "/cloudsql";
```
Comment (amanda-tarafa) [inline cloud-sql/mysql/Startup.cs:148]: Nit: remove the extra white space that was added at the end of the line, and in the next line as well. Or maybe you were trying to add example values here as well

File: cloud-sql/mysql/Startup.cs [hunk 3]
```diff
@@ -144,19 +145,19 @@ DbConnection NewMysqlUnixSocketConnection()
...
                 // The Cloud SQL proxy provides encryption between the proxy and instance. 
                 SslMode = MySqlSslMode.None,
                 // Remember - storing secrets in plaintext is potentially unsafe. Consider using
                 // something like https://cloud.google.com/secret-manager/docs/overview to help keep
                 // secrets secret.
-                Server = String.Format("{0}/{1}", dbSocketDir, instanceConnectionName),
-                UserID = Environment.GetEnvironmentVariable("DB_USER"),
-                Password = Environment.GetEnvironmentVariable("DB_PASS"),
-                Database = Environment.GetEnvironmentVariable("DB_NAME"),
+                Server = String.Format("{0}/{1}", dbSocketDir, instanceConnectionName),
```
Comment (amanda-tarafa) [inline cloud-sql/mysql/Startup.cs:157]: Nit: extra white space at the end of line or potentially missing sample value

File: cloud-sql/sql-server/Startup.cs [hunk 4]
```diff
@@ -95,10 +95,11 @@ DbConnection NewSqlServerConnection()
                 // Remember - storing secrets in plaintext is potentially unsafe. Consider using
                 // something like https://cloud.google.com/secret-manager/docs/overview to help keep
                 // secrets secret.
-                DataSource = Environment.GetEnvironmentVariable("DB_HOST"),
-                UserID = Environment.GetEnvironmentVariable("DB_USER"),
-                Password = Environment.GetEnvironmentVariable("DB_PASS"),
-                InitialCatalog = Environment.GetEnvironmentVariable("DB_NAME"),
+                DataSource = Environment.GetEnvironmentVariable("DB_HOST"),     // e.g. '127.0.0.1' 
+                                // Set Host to 'cloudsql' when deploying to App Engine Flexible environment
```
Comment (amanda-tarafa) [inline cloud-sql/sql-server/Startup.cs:99]: Nit: indent

**Review by amanda-tarafa — APPROVED** [review]
A few nits. Let me know when it's ready so I can merge.

**Comment by shubha-rajan** [comment]
@amanda-tarafa ready to merge once tests pass

**Pull request merged**